codelabs/Utopia Wholesale Codelab/Holder/0-Obtaining a credential.md

```diff
@@ -58,7 +60,7 @@ Identity credential provisioning is the process of securely issuing digital cred
 
 #### **1.1 Explore the Project Structure**
 
-First, set your project’s **“android:launchMode="singleInstance"** to prevent unnecessary recompositions, which may otherwise break the issuance process.
+First, set your project’s **`android:launchMode="singleInstance`**  in AndroidManifest.xml to prevent unnecessary recompositions, which may otherwise break the issuance process.
```

Comment:
missing closing quotation mark & I don't think this should be formatted in bold -- just as code would do, wdyt?

Reply:
code block to `AndroidManifest.xml` too?